docker-image/v1.4/debian-elasticsearch/Gemfile

```diff
@@ -10,7 +10,8 @@ gem "fluent-plugin-concat", "~> 2.3.0"
 gem "fluent-plugin-grok-parser", "~> 2.5.0"
 gem "fluent-plugin-prometheus", "~> 1.3.0"
 gem "fluent-plugin-rewrite-tag-filter", "~> 2.1.0"
-gem "fluent-plugin-elasticsearch", "~> 3.4.2"
+gem "elasticsearch", "~> 6.0"
```

Comment:
I'd have to say that it's a breaking change to change the elasticsearch version from 7.0.0 to 6.x.x even if many people are using an older version. It looks better for me to change the image name to `v1.4.2-debian-elasticsearch-2.0`.

Do you have any idea? @repeatedly

Reply:
I view it the other way around. AFAIK v1.4.2-debian-elasticsearch-1.0 was never actually released, or if it was released, was only released in the past few hours. Prior to that, [only elasticsearch 6.x plugins](https://github.com/fluent/fluentd-kubernetes-daemonset/blob/8c76f51696bdeea7643ec0e0696afebf336c45a9/docker-image/v1.3/debian-elasticsearch/Gemfile.lock#L16-L21) were used. To me, it was the unspecified use of "latest" elasticsearch pulling in the 7.0.0 plugin that was the unexpected breaking change.

Reply:
One idea is separating tags into xx-debian-elasticsearch7-1 and xx-debian-elasticsearch6-1.

Reply:
While we are at it, we should also consider that people are asking for a version that uses https://github.com/atomita/fluent-plugin-aws-elasticsearch-service

However, for now, the easiest first step is to just accept this PR as written and then add separate support for ES 7 and the AWS plugin later.